app/src/main/res/layout/new_course_item.xml

```diff
@@ -59,6 +61,7 @@
         android:paddingEnd="0dp"
```

Comment:
I think mistake is here. `right` and `end` should be equal. We should use `@dimen/extraPaddingRightCourseItem`

Reply:
Yes I have noticed that but I thought maybe there is some sense in that values

Reply:
just bug :(